core-web/libs/ui/src/lib/dot-tab-buttons/dot-tab-buttons.component.html

```diff
@@ -19,6 +19,7 @@
             <button
                 class="dot-tab__dropdown"
                 *ngIf="item.value.showDropdownButton"
+                [disabled]="item.disabled"
```

Comment:
Disanling the button brings a11y issues, the recomendation is not disable it so the user can focus and we can tell it why is not clickable.